includes/logs.inc

```diff
@@ -47,3 +56,46 @@ function islandora_find_replace_operation_logs() {
 function islandora_find_replace_get_submissions() {
   return db_query("SELECT * FROM {islandora_find_replace} WHERE state = 'complete'")->fetchAll();
 }
+
+function generate_csv(){
```

Comment:
Good. Wonder if I could better just generate those CSV directly when writing the log and then making then downloadable instead of streaming/generating them on the fly each time. But for now ++